test/integration/document_collection_test.rb

```diff
@@ -9,11 +9,33 @@ class DocumentCollectionTest < ActionDispatch::IntegrationTest
 
   test "renders metadata and document footer" do
     setup_and_visit_content_item('document_collection')
-    assert_has_component_metadata_pair("first_published", "29 February 2016")
-
-    from = ["<a href=\"/government/organisations/driver-and-vehicle-standards-agency\">Driver and Vehicle Standards Agency</a>"]
-    assert_has_component_metadata_pair("from", from)
-    assert_has_component_document_footer_pair("from", from)
+    assert_has_publisher_metadata(
+      published: "Published 29 February 2016",
+      metadata: { "From:":
+        {
+          text: "Driver and Vehicle Standards Agency",
+          href: "/government/organisations/driver-and-vehicle-standards-agency"
+        } }
+    )
+    assert_footer_has_published_dates("Published 29 February 2016")
+  end
+  
```

Comment:
The linter is complaining about trailing whitespace here https://ci.integration.publishing.service.gov.uk/job/government-frontend/job/move-document-collection-to-univeral-layout/1/console